docs/rl-algorithms/rpo.md

```diff
@@ -0,0 +1,129 @@
+# Robust Policy Optimization (RPO)
+
+## Overview
+
+RPO leverages a method of perturbing the distribution representing actions. The goal is to encourage high-entropy actions and provide a better representation of the action space. The method consists of a simple modification on top of the objective of the PPO algorithm. Assuming PPO represents action with a parameterized distribution with mean and standard deviation (e.g., Gaussian). In the RPO algorithm, the mean of the action distribution is perturbed using a random number drawn from a Uniform distribution.
+
+Original paper: 
+
+* [Robust Policy Optimization in Deep Reinforcement Learning](https://arxiv.org/abs/2212.07536)
+
+## Implemented Variants
+
+
+| Variants Implemented      | Description |
+| ----------- | ----------- |
+| :material-github: [`rpo_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/rpo_continuous_action.py), :material-file-document: [docs](/rl-algorithms/rpo/#rpo_continuous_actionpy) | For classic control tasks like Gym `Pendulum-v1`, dm_control, Pybullet. |
+
+Below are our single-file implementations of RPO:
+
+## `rpo_continuous_action.py`
+
+`rpo_continuous_action.py` works with Gym (Gymnasium), dm_control, Pybullet environments with continuous action and vector observations.
+
+The [rpo_continuous_action.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/rpo_continuous_action.py) has the following features (similar to [ppo_continuous_action.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_continuous_action.py)):
+
+* For continuous action space. Also implemented Mujoco-specific code-level optimizations
+* Works with the `Box` observation space of low-level features
+* Works with the `Box` (continuous) action space
+* adding experimental support for [Gymnasium](https://gymnasium.farama.org/)
+* 🧪 support `dm_control` environments via [Shimmy](https://github.com/Farama-Foundation/Shimmy)
+
+### Usage
+
+```bash
+# mujoco v4 environments
+poetry install --with mujoco
+python cleanrl/rpo_continuous_action.py --help
+python cleanrl/rpo_continuous_action.py --env-id Hopper-v2
+# dm_control v4 environments
+poetry install --with mujoco,dm_control
+python cleanrl/rpo_continuous_action.py --env-id dm_control/cartpole-balance-v0
+# backwards compatibility with mujoco v2 environments
+poetry install --with mujoco_py,mujoco
+python cleanrl/rpo_continuous_action.py --env-id Hopper-v2
+```
+
+### Explanation of the logged metrics
+
+See [related docs](/rl-algorithms/ppo/#explanation-of-the-logged-metrics) for `ppo.py`.
+
+### Implementation details
+Similar to PPO [ppo_continuous_action.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_continuous_action.py) with a change in `get_action_and_value` method in `Agent` class.
+
+[TODO: Add code snippets, and comparison with PPO]
+
```

Comment:
In the implementation details section, you should also add a note on the difference on hyperparameters. We used a table for `ppo_continuous_action_isaacgym.py`, but maybe in this case you just add a note since there is only one hyperparameter difference (--total-timesteps=8000000)

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/5555347/209713230-11658d88-fed1-49d7-a8c1-c9555e7d3546.png">